support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -53,11 +53,19 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
   private def sendAcquisitionEvent(state: SendAcquisitionEventState, requestInfo: RequestInfo, services: Services) = {
     sendPaymentSuccessMetric(state)
 
-    services.bigQueryService.tableInsertRow(
-      AcquisitionDataRowBuilder.buildFromState(state, requestInfo)
-    ).value.flatMap {
+    val acquisition = AcquisitionDataRowBuilder.buildFromState(state, requestInfo)
+
+    val streamFuture = services.acquisitionsStreamService.putAcquisition(acquisition)
+    val biqQueryFuture = services.bigQueryService.tableInsertRow(acquisition)
+
+    val result = for {
+      streamResult <- streamFuture
+      bigQueryResult <- biqQueryFuture
+    } yield ()
+
+    result.value.map {
       case Left(errorMessage) => throw new RetryUnlimited(errorMessage)
```

Comment:
I think we should change this to `RetryNone`. An alarm should go off if it fails

Reply:
ooh that's really bad, I don't think we thought about that one at the time. I agree with retry None.

Reply:
Actually - I've just noticed this morning that the `SendAcquisitionEvent` failed because of an issue on BigQuery's side, and succeeded on the retry.
So it seems we do want to retry, but I'm not sure how to do this besides splitting BigQuery and kinesis into separate lambdas, which would be a shame

Reply:
another option is to just retry in scala if we think the error would be simple and transient
The benefit of step functions retries is the lambda can terminate between each ones so it can go until zuora comes back up rather than just 15 minutes, and also aany fatal errors that cause the jvm to quit are alaso covered.  But that may not apply for this situation?

Reply:
That's a good idea @johnduffell 
Let's implement retry logic in the scala for both bigquery and kinesis, and make this lambda return RetryNone when it fails.

Reply:
Also, ideally that retry logic can be shared with payment-api